v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/options/SchemaRegistryOptions.java

```diff
@@ -188,4 +189,52 @@ public interface SchemaRegistryOptions extends PipelineOptions {
   String getSchemaRegistryKeyPasswordSecretId();
 
   void setSchemaRegistryKeyPasswordSecretId(String keyPasswordSecretId);
+
+  @TemplateParameter.Text(
+      order = 11,
+      parentName = "schemaRegistryAuthenticationMode",
+      parentTriggerValues = {KafkaAuthenticationMethod.OAUTH},
+      optional = true,
+      description = "Client ID",
+      helpText =
+          "Client ID used to obtain client credentials grant from OAuth/OIDC identity provider.")
+  String getSchemaRegistryClientId();
```

Comment:
The parameter name doesn't make it clear that this is the ClientID to be used with OAuth, it sounds like "the id of the schema registry client", which is confusing.
I'd add "OAuth" prefix to all of the new params, e.g. `schemaRegistryOAuthClientId`, `schemaRegistryOAuthTokenEndpointUrl`, etc.

Reply:
Note: it's not a problem in the UI, but could be a problem when running the template via CLI and/or reading the list of parameters in the docs.

Reply:
Done.